src/api/app/helpers/webui/package_helper.rb

```diff
@@ -72,4 +72,13 @@ def nbsp(text)
     # But at this point we know it truly is html safe
     result.html_safe
   end
+
+  def humanize_time(secs)
```

Comment:
Because we are not showing a specific datetime, and we want the duration of the build and not since when it was done.